### DISCUSSION TITLE
assign me to a cryptographic issue please

### DISCUSSION THREAD

**Discussion by gogo2464** [freenet/freenet-core#977] [General]
Hello.

Could you find a easy first issue related to cryptography please? I need to progress in crypto. I just finish some pr and I will rush it!

**Comment by iduartgomez** [comment]
This doesn't belong in issues, better suited for Matrix tbh, but moved to discussions.

I don't think there are any pending crypto issues right now tbh, maybe when one opens up I'll let you know.

**Reply by gogo2464** [reply]
I would like to code crypto in freenet network directly. if possible

**Reply by gogo2464** [reply]
but thanks for letting me know

**Comment by gogo2464** [comment]
the freenet publishing has been reported. Could I be assigned to a crypto/network issue please?
